lint.sh

```diff
@@ -20,6 +20,7 @@ if [ "$LINT" == true ]; then
         statsmodels/info.py \
         statsmodels/resampling/ \
         statsmodels/interface/ \
+        statsmodels/examples/tests/ \
```

Comment:
There's a typo two lines down that needs fixing.  A new PR from you will get through before any of the existing PRs that fix it.  Also statsmodels/info.py above no longer exists.